package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "loopback-context",
-  "version": "3.0.0-alpha.1",
```

Comment:
Please don't change the version number, we will bump it when releasing a new version.